docs/conceptual/array.sumby['t,^u]-function-[fsharp].md

```diff
@@ -30,29 +30,41 @@ Array.sumBy projection array
 ```
 
 #### Parameters
-*projection*
-Type: **'T -&gt; ^U**
-
+*projection* Type: **'T -&gt; ^U**
```

Comment:
Would also still keep the previous version, as it's cleaner when it's separated in two lines.